modules/loader/src/main/java/com/opengamma/strata/loader/csv/TradeCsvInfoResolver.java

```diff
@@ -433,8 +433,28 @@ public default CdsIndexTrade parseCdsIndexTrade(CsvRow row, TradeInfo info) {
     return CdsTradeCsvPlugin.parseCdsIndex(row, info, this);
   }
 
+  //-------------------------------------------------------------------------
   /**
-   * Parses any other kind of trade from CSV.
+   * Parses any kind of trade from CSV before standard matching.
+   * <p>
+   * This is called before the standard matching on the 'Product Type' column.
+   * As such, it allows the standard parsing to be replaced for a given type.
+   * 
+   * @param typeUpper  the upper case product type column
```

Comment:
Has 2 spaces after the parameter name always been a thing?

Reply:
In Strata, yes

Reply:
Got it 👍